### DISCUSSION TITLE
User should be able to create a comment on comment

### DISCUSSION THREAD

**Discussion by R3dIO** [RTX2080-UF/gatorshare-frontend#84] [Errors]
Implement functionality to comment on other comments on front-end as on backend we can just use the comment API with parent Id set as the comment Id on which the child comment is made.

**Comment by R3dIO** [comment]
@tatirajurishabh For this functionality what will you be more comfortable with 1) I send you to array containing all child comments and you build the comment hierarchy tree in the frontend or 2) I send you the tree in form of nested JSON and you just write the logic of displaying it?

**Comment by rtdtwo** [comment]
@R3dIO Ideally the second option, nested comments inside the JSON itself.

**Comment by R3dIO** [comment]
Got you! will start implementing it and update the current issue with relevant details.
